src/main/java/exception/MovieNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package exception;
+
+public class MovieNotFoundException extends RuntimeException {
```

Comment:
there is no need to create custom exception for every possible case, let's just use runtime exception